apps/andi/lib/andi/input_schemas/dataset.ex

```diff
@@ -0,0 +1,292 @@
+defmodule Andi.InputSchemas.Technical do
```

Comment:
This file feels massive. Haven't done much Ecto, but is there a way to break this up?